utils/defaults.stanza

```diff
@@ -65,17 +65,37 @@ public pcb-rules default-rules :
   min-pth-pin-solder-clearance = 3.0
 set-default-rules(default-rules)
 
+public pcb-via default-via :
+  name = "Default TH"     
+  start = LayerIndex(0, Top)
+  stop = LayerIndex(0, Bottom)
+  diameter = 0.6
+  hole-diameter = 0.3
+  type = MechanicalDrill
```

Comment:
May want to add tented = true by default (I think that most vias are tented by default?)

Reply:
https://jlcpcb.com/help/article/122-Via-Covering:-Tented,-Untented,-Plugged,-Epoxy-Filled-and-Copper-epoxy-filled 

tldr:
Tented Vias
This option is the most commonly used. Vias are covered with soldermask, and surface finish such as HASL is not applied to vias.

Reply:
If that isn't the default value for a pcb-via then we should make it the default value. 

This PR is canonical, reality needs to be updated.